Formula/android-sdk.rb

```diff
@@ -117,6 +125,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    assert_match version.to_s, shell_output("#{prefix}/tools/emulator -version")
```

Comment:
It's downloaded separately as part of the `build-tools` so it's quite possible that it will change at some point. Google has a bad habit of doing that...

It currently outputs
```
Android Debug Bridge version 1.0.36
Revision 9f299f282ec9-android
```
Should I check that it says "Android Debug Bridge" in the output?